src/style.css

```
@@ -1,3 +1,62 @@
 body {
   margin: 0;
 }
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
```

```suggestion
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```

Same but a bit shorter

---

Still not fixed from the previous review